sling.go

```diff
@@ -472,10 +565,14 @@ func (s *Sling) do(req *http.Request) (*http.Response, error) {
 // If the status code of response is 204(no content) or the Content-Length is 0,
 // decoding is skipped. Any error sending the request or decoding the response
 // is returned.
-func (s *Sling) doDecode(req *http.Request, successV, failureV interface{}) (*http.Response, error) {
+func (s *Sling) doDecode(req *http.Request, successV, failureV interface{}) (*Response, error) {
+	if successV == nil && failureV == nil {
```

Comment:
Since this function is designed for decoding, it doesn't really make sense to allow it to be called when nothing is being decoded.